ucm2/Allwinner/A64/PinePhone/VoiceCall.conf

```diff
@@ -0,0 +1,149 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Switch' off"
+		cset "name='Headphone Source Playback Route' DAC"
+		cset "name='Line In Playback Switch' off"
+		cset "name='Line Out Playback Switch' off"
+		cset "name='Line Out Source Playback Route' Mono Differential"
+		cset "name='Mic1 Playback Switch' off"
+		cset "name='Mic2 Playback Switch' off"
+		cset "name='AIF1 DA0 Playback Volume' 160"
+		cset "name='AIF2 DAC Playback Volume' 160"
+		cset "name='AIF3 ADC Source Capture Route' None"
+		cset "name='AIF2 DAC Source Playback Route' None"
+		cset "name='DAC Playback Switch' on"
+		cset "name='DAC Playback Volume' 160"
+		cset "name='ADC Digital DAC Playback Switch' off"
+		cset "name='AIF1 Slot 0 Digital DAC Playback Switch' on"
```

Comment:
AIF1 Slot 0 is the CPU audio interface. Is it intentional that this is enabled during a call?

Reply:
Perhaps folks want to record their calls? Or maybe GNOME Calls uses the VoiceCall even for SIP?

What's the disadvantage of leaving this enabled?

Reply:
I suppose there isn't any downside. This won't cause any additional hardware to be powered up, since the capture path must be enabled anyway for recording.

One note though on call recording: you will need to enable "AIF2 Digital ADC Capture Switch" in order to capture audio from the modem. ("AIF1 Data Digital ADC Capture Switch" only gets you audio from the analog codec, i.e. the microphone).

Reply:
Done.